src/style.scss

```diff
@@ -4,89 +4,129 @@
 // ---------------------------------------------------------------------
 
 // Import Auro tokens
-@import "./node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables";
-@import "./node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties";
-@import "./node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariableMap";
+@import './node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+@import './node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+@import './node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariableMap';
 
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/breakpoints";
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/core";
-
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/utilityClasses/displayProperties";
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/breakpoints';
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/core';
 
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/utilityClasses/displayProperties';
 $auro-inset-spacing-options: lg;
-$auro-inset-directions:'';
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/utilityMixins/insetUtility";
+$auro-inset-directions: '';
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/utilityMixins/insetUtility';
 
 ::slotted(*:hover) {
   cursor: pointer;
 }
 
+:host(:not([data-show])) {
+  .popover {
+    @extend .util_displayHiddenVisually;
+  }
+}
+
+:host([data-show]) {
+  .popover {
+    z-index: var(--auro-depth-tooltip);
+
+    background-color: var(--auro-color-background-lightest);
+  }
+}
+
 :host([addSpace]) {
   .popover {
     margin: 1rem 0 !important;
   }
 }
 
 .popover {
-  border-radius: var(--auro-border-radius);
-  background: var(--auro-color-background-lightest);
-  box-shadow: -2px 0px 5px 2px rgba(0, 0, 0, 0.08),
-    0px 2px 5px 1px rgba(0, 0, 0, 0.08);
   display: inline-block;
+
```

Comment:
There's a lot of extra newlines and reordering with this PR -- is this aligned with Auro's CSS guidelines?

Reply:
This is just grouping relevant styles for readability. e.g....

```
    position: absolute;
    bottom: -20px;
    left: 0;

    display: block;

    width: 100%;
    height: 20px;

    content: '';
```

Is much more logically readable than this

```
    display: block;
    position: absolute;
    content: '';
    bottom: -20px;
    left: 0;
    width: 100%;
    height: 20px;

```

Consistent patterning and grouping of the properties makes it a lot easier to read through and parse out what is going on. Also, once the SCSS compiles all the spacing is stripped out anyhow. So it only affects human readability of the code.

Reply:
Opened a thread for all of us to talk this one out.

Reply:
Based on the conversation in Teams, the resolution to this is that this is not a bad pattern, but if this pattern is to be used in a file, that pattern needs to be always used in that file.